### PR TITLE
shirley / s1332 update status action

### PR DESCRIPTION
**Pull request by shirleygordon-blink**

### CODE REVIEW

File: mask.yaml [hunk 1]
```diff
@@ -70,6 +70,35 @@ actions:
...
+        alias: "Accept"
+        required: true
+        default: application/vnd.pagerduty+json;version=2
+        description: The `Accept` header is used as a versioning header.
+        index: 3
+#      Content-Type:
+#        alias: "Content-Type"
+#        required: true
+#        default: application/json
+#        index: 2
```
Comment (ozhey) [inline mask.yaml:97]: Why do we have this commented out?

File: mask.yaml [hunk 2]
```diff
@@ -70,6 +70,35 @@ actions:
...
+        required: true
+        description: The email address of a valid user associated with the account making the request.
+        type: email
+        index: 2
+      Accept:
+        alias: "Accept"
+        required: true
+        default: application/vnd.pagerduty+json;version=2
+        description: The `Accept` header is used as a versioning header.
+        index: 3
```
Comment (ozhey) [inline mask.yaml:92]: I think this is only if you use OAuth and is already handled by the setCustomAuthHeaders function. Take a look at the main.go file, line 110.

File: mask.yaml (hunk 1 above)
Comment (shirleygordon-blink) [reply mask.yaml:97]: oops forgot to delete it, it's unnecessary because the content type header is added automatically

File: mask.yaml (hunk 2 above)
Comment (shirleygordon-blink) [reply mask.yaml]: right, fixed

**Review by ozhey — APPROVED** [review]
Thanks!

**Pull request merged**